tests/framework/Optimizers/GeneticAlgorithms/continuous/unconstrained/test_ensemble_withGA.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers/GeneticAlgorithms/test_ensemble_withGA</name>
```

Comment:
classesTested node is required. Could you add that?

Reply:
fixed!